tests/dummy/app/deprecation-workflow.js

```diff
@@ -0,0 +1,11 @@
+import setupDeprecationWorkflow from 'ember-cli-deprecation-workflow';
+
+setupDeprecationWorkflow({
+  workflow: [
+    { handler: 'throw', matchId: 'this-property-fallback' },
+    { handler: 'throw', matchId: 'remove-owner-inject' },
+    { handler: 'throw', matchId: 'deprecate-auto-location' },
+    { handler: 'throw', matchId: 'ember-string.add-package' },
+    { handler: 'throw', matchId: 'ember-string.from-ember-module' },
```

Comment:
Upgrading ember identified a few other deprecations. This PR fixes them and adds them to this workflow file to confirm that they are fixed.